model/src/main/java/io/gravitee/definition/model/flow/FlowStage.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.definition.model.flow;
+
+/**
+ * In which stage the flow is configured
+ * @author Yann TAVERNIER (yann.tavernier at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public enum FlowStage {
+    PLATFORM,
+    PLAN,
+    API,
+    UNDEFINED,
```

Comment:
`UNDEFINED` is still used ?

Reply:
good catch, I just FP
I will have to do the cleaning in front part too